policyengine_us/variables/gov/states/mo/tax/income/credits/property_tax/mo_ptc_taxunit_eligible.py

```diff
@@ -14,37 +14,32 @@ class mo_property_tax_credit_demographic_tests(Variable):
     defined_for = StateCode.MO
 
     def formula(tax_unit, period, parameters):
-        # Eligibility
-        # Check for age eligiblity
+        # check age
         age_head = tax_unit("age_head", period)
         age_spouse = tax_unit("age_spouse", period)
-        age_threshold = parameters(
-            period
-        ).gov.states.mo.tax.income.credits.property_tax.age_threshold
-        elderly_head = age_head >= age_threshold
-        elderly_spouse = age_spouse >= age_threshold
+        p = parameters(period).gov.states.mo.tax.income.credits.property_tax
+        elderly_head = age_head >= p.age_threshold
+        elderly_spouse = age_spouse >= p.age_threshold
         elderly_head_or_spouse = elderly_head | elderly_spouse
-
-        # Check for disability eligibility
+        # check disability
         disabled_head = tax_unit("disabled_head", period)
         disabled_spouse = tax_unit("disabled_spouse", period)
         disabled_head_or_spouse = disabled_head | disabled_spouse
-
-        # Check for military disabled eligibility
+        # check for military disability
         military_disabled_head = tax_unit("military_disabled_head", period)
         military_disabled_spouse = tax_unit("military_disabled_spouse", period)
         military_disabled_head_or_spouse = (
             military_disabled_head | military_disabled_spouse
         )
-
-        # Check for receipt of surviving spouse benefits
-        receives_survivor_benefits = (
-            add(tax_unit, period, ["social_security_survivors"]) > 0
-        )
-
+        # check social security survivor benefits eligibility
+        survivor_ben = add(tax_unit, period, ["social_security_survivors"]) > 0
+        aged_head = age_head >= 60
```

Comment:
Could you make 60 a parameter?

Quote from [legal code](https://revisor.mo.gov/main/OneSection.aspx?section=135.010&bid=6435) for posterity 
>or if the claimant has reached the age of sixty on or before the last day of the calendar year

Reply:
Done